host/xtest/regression_1000.c

```diff
@@ -2036,3 +2037,190 @@ static void xtest_tee_test_1025(ADBG_Case_t *c)
 }
 ADBG_CASE_DEFINE(regression, 1025, xtest_tee_test_1025,
 		 "Test memref NULL and/or 0 bytes size");
+
+#define TEE_UUID_NS_NAME_SIZE  128
+
+/*
+ * TEE Client UUID name space identifier (UUIDv4)
+ *
+ * Value here is random UUID that is allocated as name space identifier for
+ * forming Client UUID's for TEE environment using UUIDv5 scheme.
+ */
+static const char *client_uuid_linux_ns = "58ac9ca0-2086-4683-a1b8-ec4bc08e01b6";
+
+/* TEEC_LOGIN_PUBLIC's Client UUID is NIL UUID */
+static TEEC_UUID client_uuid_public = { };
+
+static TEEC_Result get_client_identity(TEEC_Session *session, uint32_t *login,
+				       TEEC_UUID *client_uuid)
+{
+	TEEC_Operation operation = { };
+	TEEC_Result result = TEEC_ERROR_GENERIC;
+
+	operation.params[1].tmpref.buffer = client_uuid;
+	operation.params[1].tmpref.size = sizeof(*client_uuid);
+
+	operation.paramTypes = TEEC_PARAM_TYPES(TEEC_VALUE_OUTPUT,
+						TEEC_MEMREF_TEMP_OUTPUT,
+						TEEC_NONE, TEEC_NONE);
+
+	result = TEEC_InvokeCommand(session, TA_OS_TEST_CMD_CLIENT_IDENTITY,
+				    &operation, NULL);
+
+	if (result != TEEC_SUCCESS)
+		return result;
+
+	*login = operation.params[0].value.a;
+
+	return TEEC_SUCCESS;
+}
+
+static void xtest_tee_test_1026(ADBG_Case_t *c)
+{
+	TEEC_Result result = TEEC_ERROR_GENERIC;
+	uint32_t ret_orig = 0;
+	TEEC_Context context = { };
+	TEEC_Session session = { };
+	uint32_t login = (uint32_t)-1;
+	TEEC_UUID client_uuid = { };
+
+	result = TEEC_InitializeContext(NULL, &context);
+	if (!ADBG_EXPECT_TEEC_SUCCESS(c, result))
+		return;
+
+	result = TEEC_OpenSession(&context, &session, &os_test_ta_uuid,
+				  TEEC_LOGIN_PUBLIC, NULL, NULL, &ret_orig);
```

Comment:
As mentioned by @jenswi-linaro below, remove this and use `xtest_teec_ctx`  at line 2091.

```c
       result = TEEC_OpenSession(&xtest_teec_ctx, &session,
				 &pta_invoke_tests_ta_uuid,
				 NULL, &ret_orig);
	if (!ADBG_EXPECT_TEEC_SUCCESS(c, result))
		return;

	result = get_client_identity(&session, &login, &client_uuid);
	if (!ADBG_EXPECT_TEEC_SUCCESS(c, result))
		goto out;

	ADBG_EXPECT_COMPARE_UNSIGNED(c, login, ==, TEEC_LOGIN_PUBLIC);

	ADBG_EXPECT_EQUAL(c, &client_uuid_public, &client_uuid,
			  sizeof(TEEC_UUID));

out:
	TEEC_CloseSession(&session);
}
```

And same inside `xtest_tee_test_1027()` and `xtest_tee_test_1028()` below.

Reply:
Updated.